schemas/alpha/azure.yaml.json

```diff
@@ -42,8 +42,7 @@
                     "title": "Type of infrastructure provisioning provider",
                     "description": "Optional. The infrastructure provisioning provider used to provision the Azure resources for the application. (Default: bicep)",
                     "enum": [
-                        "bicep",
-                        "terraform"
+                        "bicep"
```

Comment:
Please undo the change here - we want the alpha schema to be a superset of the released schema (that way if someone points at the alpha schema, they see everything that is also GA'd).